frontend/src/components/modalContent/AddReportModal.js

```diff
@@ -27,4 +29,7 @@ export const AddReportModal = (props) => (
 AddReportModal.propTypes = propTypes;
 AddReportModal.defaultProps = defaultProps;
 
-export default AddReportModal;
+export default connect(
+  null,
+  null
+)(AddReportModal);
```

Comment:
I'm not an expert on `connect`, but this looks unnecessary or like some sort of anti-pattern.

Reply:
Right, this is outdated - I was checking to see if there was an issue where a component in a modal couldn't connect to the store if the modal wasn't connected.

Removed.